projects/vdk-control-cli/tests/vdk/internal/control/command_groups/job/test_download_job.py

```diff
@@ -82,3 +99,32 @@ def test_download_job_does_not_exist(httpserver: PluginHTTPServer, tmpdir: Local
     assert "The requested resource cannot be found" in result.output
     # assert that vdk does not try cleaning up a non-existent archive
     assert "Cannot cleanup archive" not in result.output
+
+
+def test_download_key_does_error(httpserver: PluginHTTPServer, tmpdir: LocalPath):
```

Comment:
nit: the name of the test could me a bit more descriptive - what kind of error is raised for example.

Reply:
Here we do not check what the error is, we check whether the operation returns success (which is the expected. behaviour since the download job is done) and leaves a warning with the error about the keytab download
I will leave it as it is